pkg/param/param.go

```diff
@@ -513,7 +513,7 @@ func UpdatePhaseParams(ctx context.Context, tp *TemplateParams, phaseName string
 
 // UpdateDeferPhaseParams updates the TemplateParams deferPhase output with passed output
 // This output would be generated/passed by execution of the phase
-func UpdateDeferPhaseParams(ctx context.Context, tp *TemplateParams, output map[string]interface{}) {
```

Comment:
This change probably shouldn't belong to this PR.

Reply:
Just removed unused parameter

Reply:
Yes, but this makes the PR harder to read. Also, this is an exported function. And may be used by those who use kanister as a library. I'd better not change it at all. But even if it has to be changed, I think it deserves its own PR.

Reply:
Ok, reverted it back